views/entry/success.tt

```diff
@@ -24,8 +24,14 @@ the same terms as Perl itself.  For a copy of the license, please reference
 [%- END -%]
 
 [%- IF poststatus.ml_string != ".edit.delete" -%]
+
 <p>[% extradata.security_ml | ml %]</p>
-<p>[% ".extradata.subject" | ml( subject => extradata.subject) %]</p>
+<p>[% ".extradata.subj" | ml %]
+    [% IF extradata.subject.length > 0 %]
+        [% ".extradata.subject.subject" | ml( subject => extradata.subject ) %]
```

Comment:
Here is what I recommend instead for your line 31:
```
[% extradata.subject | html %]
```
The `| html` does the same thing LJ::ehtml would do in the controller, and as I said elsewhere, there's no need to pass this through the translation system at all.